velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -699,5 +699,26 @@ TEST_F(ArithmeticTest, truncate) {
   EXPECT_DOUBLE_EQ(truncate(123456789012345678901.23, -21).value(), 0.0);
 }
 
+TEST_F(ArithmeticTest, BetaCDF) {
```

Comment:
Nit : test name "BetaCDF" should start with a lowercase letter.

Reply:
Fixed it.